src/components/nested-sortable/components/node-draggable/NodeDraggableTarget.jsx

```diff
@@ -32,13 +33,19 @@ const nodeTarget = {
     const hoverId = props.item.id;
     const hoverParentId = props.parentId;
     const doesParentAllowDrop = props.doesParentAllowDrop;
+    const doesAllowDrop = props.item.doesAllowDrop;
 
     // Don't look for every target container, only first, non-greedy
     const isOverCurrent = monitor.isOver({ shallow: true });
     if (!isOverCurrent) {
       return;
     }
 
+    // don't allow drop if doesAllowDrop is false
+    if (!_.isUndefined(doesAllowDrop) && !doesAllowDrop) {
```

Comment:
I'd prefer`!!doesAllowDrop && !doesAllowDrop)`

Reply:
sure, I wanted to be sure it is undefined (undefined is when you didn't pass that property) but it is ok to use !! as this can be only boolean